defy-labs/blackmarket
# feat(components): welcome modal

### Description
Adds a welcome modal to pages (via NavBar component) and saves the view state in Local Storage so that the modal is only presented to the user once.

NOTE: Includes translations for en and ja locales

### How to test
- Load any page on the site for the first time (try with incognito tab) and confirm the modal is shown to the user, and displays the correct translation based on the site language setting.
- After closing the modal, reload the page and confirm that the modal is no longer shown to the user.
